### PR TITLE
Update rabbit keys

### PR DESCRIPTION
**Pull request by joepvd**
No warnings/errors here: https://staging.travis-ci.org/joepvd/travis-staging-test/jobs/736659#L239

### CODE REVIEW

File: lib/travis/build/appliances/update_apt_keys.rb [hunk 1]
```diff
@@ -22,8 +22,27 @@ def apply
...
             KEYUPDATE
             sh.cmd command, echo: false
+
+            sh.cmd rabbit_key_update, echo: false
           end
         end
+
+        def rabbit_key_update
+          <<~EORABBIT
+          if [[ ( \"$TRAVIS_DIST\" == trusty || \"$TRAVIS_DIST\" == precise) && -f /etc/init.d/rabbitmq-server ]]; then
```
Comment (cotsog) [inline lib/travis/build/appliances/update_apt_keys.rb]: Just to confirm, `precise` and `trusty` both use the `rabbitmq-trusty` key URL?

File: lib/travis/build/appliances/update_apt_keys.rb [hunk 2]
```diff
@@ -22,8 +22,27 @@ def apply
...
+            curl -sSL #{key_url('rabbitmq-trusty')} | sudo apt-key add -
```
Comment (cotsog) [inline lib/travis/build/appliances/update_apt_keys.rb]: Is `rabbitmq-trusty` added via another PR e.g. keychain?

File: lib/travis/build/appliances/update_apt_keys.rb (hunk 1 above)
Comment (joepvd) [reply lib/travis/build/appliances/update_apt_keys.rb]: Confirm. The contents of the following files is the same:
https://github.com/travis-ci/apt-source-safelist/blob/master/keys/rabbitmq-trusty.asc
https://github.com/travis-ci/apt-source-safelist/blob/master/keys/rabbitmq-precise.asc

File: lib/travis/build/appliances/update_apt_keys.rb (hunk 2 above)
Comment (joepvd) [reply lib/travis/build/appliances/update_apt_keys.rb]: Yes, https://github.com/travis-ci/apt-source-safelist/pull/397 merged the updated keys.

**Pull request merged**